client/python/README.md

```diff
@@ -9,31 +9,107 @@ pip install djclient
 ## Examples
 
 To initialize the client:
-```
-from djclient import DJClient
+```python
+from djclient.dj import DJClient, Source, Dimension, Transform, Metric
 client = DJClient("http://dj-endpoint:8000")
 ```
 
+### Catalogs and Engines
+
 To list available catalogs for the DJ host:
-```
+```python
 client.catalogs()
```

Comment:
I'd find it clearer if we instantiate the client as `dj = DJClient("http://dj-endpoint:8000")`, then this line would become `dj.catalogs()`. What do you think?